packages/main/src/webComponents/Select/index.tsx

```diff
@@ -23,6 +23,13 @@ interface SelectAttributes {
    * **Note:** A disabled component is noninteractive.
    */
   disabled?: boolean;
+  //todo adjust types and description + add story
+  /**
+   * Defines a reference (ID or DOM element) of component's menu of options. as alternative to define the select's dropdown.
+   *
+   * **Note:** Usage of `SelectMenu` is recommended.
+   */
+  menu?: string | HTMLElement;
```

Comment:
should be only of type `string` here, the `string | HTMLElement` type should move to the `SelectDomRef`